resources/metadata/whitesource.yaml

```diff
@@ -0,0 +1,523 @@
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: ./build.sbt
+      conditions:
+      - conditionRef: strings-equal
+        params:
+        - name: scanType 
+          value: sbt
+    - name: buildDescriptorFile
+      type: string
+      description: Explicit path to the build descriptor file.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: ./dub.json
+      conditions:
+      - conditionRef: strings-equal
+        params:
+        - name: scanType
+          value: dub
+    - name: createProductFromPipeline
+      type: bool
+      description: Whether to create the related WhiteSource product on the fly based on the supplied pipeline configuration.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: true
+    - name: securityVulnerabilities
+      type: bool
+      description: Whether security compliance is considered and reported as part of the assessment.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: true
+    - name: timeout
+      type: string
+      description: Timeout in seconds until a HTTP call is forcefully terminated.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: 0
+    - name: agentDownloadUrl
+      type: string
+      description: URL used to download the latest version of the WhiteSource Unified Agent.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: https://github.com/whitesource/unified-agent-distribution/releases/latest/download/${config.agentFileName}
+    - name: configFilePath
+      type: string
+      description: Explicit path to the WhiteSource Unified Agent configuration file.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: ./wss-unified-agent.config
+    - name: vulnerabilityReportFileName
+      type: string
+      description: Name of the file the vulnerability report is written to.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: piper_whitesource_vulnerability_report
+    - name: parallelLimit
+      type: string
+      description: 'Limit of parallel jobs being run at once in case of `scanType:
+        ''mta''` based scenarios, defaults to `15`.'
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: 15
+    - name: reporting
+      type: bool
+      description: Whether assessment is being done at all, defaults to `true`.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: true
+    - name: whitesource/serviceUrl
+      type: string
+      description: URL to the WhiteSource server API used for communication, defaults to `https://saas.whitesourcesoftware.com/api`.
```

Comment:
Do not describe defaults, if one is defined it's automatically printed in the docs (`*.md`).

```suggestion
      description: URL to the WhiteSource server API used for communication.
```